test/integration-tests/aspnet/IntegrationTests.AspNet/AspNetTests.cs

```diff
@@ -43,4 +44,36 @@ public async Task SubmitsTraces()
             Assert.True(spans.Count >= 1, $"Expecting at least 1 span, only received {spans.Count}");
         }
     }
+
+    [Fact]
+    [Trait("Category", "EndToEnd")]
+    [Trait("Containers", "Windows")]
+    public async Task SubmitsMetrics()
+    {
+        var prometheusPort = TcpPortProvider.GetOpenPort();
+        var webPort = TcpPortProvider.GetOpenPort();
+
+        using (var container = await StartContainerForMetricsAsync(webPort, prometheusPort))
+        {
+            var client = new HttpClient();
+
+            var response = await client.GetAsync($"http://localhost:{webPort}/metrics");
+            var content = await response.Content.ReadAsStringAsync();
+
+            Output.WriteLine("Sample response:");
+            Output.WriteLine(content);
+
+            response = await client.GetAsync($"http://localhost:{webPort}/prometheusmetrics");
+            content = await response.Content.ReadAsStringAsync();
+
+            Output.WriteLine("Raw metrics from Prometheus:");
+            Output.WriteLine(content);
+
+            //PrometheusMetrics
+
+            Assert.Equal(HttpStatusCode.OK, response.StatusCode);
+            Assert.Contains("MyFruitCounter{color=\"red\",name=\"apple\"} 6", content);
+            Assert.Contains("http_server_duration_ms_count{http_method=\"GET\",http_scheme=\"http\",http_status_code=\"200\"}", content);
```

Comment:
Want to take feedback on this approach of asserting. If this looks fine, will refactor the PR and mark it for review.

In dotnet OpenTelemetry, asserting is done using similar approach. Ref: https://github.com/open-telemetry/opentelemetry-dotnet/blob/cc5083e37bbe87cbb94aa9a34cbb7bf3a149f7a3/test/OpenTelemetry.Exporter.Prometheus.Tests/PrometheusExporterHttpServerTests.cs#L167

Reply:
I prefer the way done here: https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/716, because:

1. I think we should use OTLP for integration testing of the instrumentations
2. We do not need to make such detailed assertions - we just need to check if the instrumentation from the SDK is working 
3. Uses FluentAssertions